pages/lesson/[id]/index.js

```diff
@@ -117,20 +117,39 @@ export default function lessonIntro() {
                 </Grid>
                 <Grid item lg={3} sx={{ pl: 3, borderLeft: 1, minHeight: '100vh' }}>
                     <Grid container direction="column" sx={{ top: 35, position: 'sticky' }}>
-                        {/* <Grid item sx={{ mb: 5 }}>
+                
```

Comment:
Won't be needing this file anymore. Search bar code has been moved into its own component